src/utils/client.ts

```diff
@@ -347,6 +347,7 @@ export default class Client {
   submitAuthInfo({ authId, authPhone = '', authEmail = '' }: any) {
     const { userId } = initInfo();
     initState.authId = authId;
+    initState.loginRedirectFlag = true;
```

Comment:
are there any race conditions here

Reply:
the renderCb below handles the race condition between authentication and form navigation